reports/views.py

```diff
@@ -56,6 +61,40 @@ def post(self, request, *args, **kwargs):
         return Response(resp, status=status)
 
 
+class MSISDNMessagesReportView(APIView):
+    """
+    Generate a report of MSISDNs and the messages they've received.
+
+    POST: Start generating a report.
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, *args, **kwargs):
+        request_data = request.data.copy()
+
+        request_data['start_date'] = '2017-09-01'
+        request_data['end_date'] = '2018-09-01'
```

Comment:
I was thinking we could make this slightly more generic and the user could specify the dates. Would that complicate things?